### PR TITLE
Remove empty xml comment lines

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Instead of #7356.

Please review commit by commit.

- The PR only remove _empty_ public comment XML lines. Please don't request other changes.
- The changes were made in Notepad++ by regex.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/ReadConsoleCmdlet.cs [hunk 1]
```diff
@@ -14,7 +14,6 @@
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    ///
     /// Retrieves input from the host virtual console and writes it to the pipeline output.
     ///
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/ReadConsoleCmdlet.cs]: The regular expression should remove this whitespace unless the line after the whitespace contains text and not `<`.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/ReadConsoleCmdlet.cs (hunk 1 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.Commands.Utility/commands/utility/ReadConsoleCmdlet.cs]: Fixed.

File: src/Microsoft.PowerShell.Security/security/CertificateProvider.cs [hunk 2]
```diff
@@ -868,30 +853,24 @@ protected override object RemoveItemDynamicParameters(string path, bool recurse)
...
-        ///
         /// <returns>
         /// Nothing.  The new certificate store object is
         /// written to the context's pipeline.
         /// </returns>
-        ///
         /// <exception cref="System.ArgumentException">
         ///     path is null or empty.
         /// </exception>
         ///
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.Security/security/CertificateProvider.cs]: Missed this one

File: src/System.Management.Automation/FormatAndOutput/common/ILineOutput.cs [hunk 3]
```diff
@@ -153,8 +153,6 @@ internal abstract class LineOutput
         /// start the playback
         /// </summary>
         internal virtual void ExecuteBufferPlayBack(DoPlayBackCall playback) { }
-
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/FormatAndOutput/common/ILineOutput.cs]: This whitespace should probably stay

File: src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs [hunk 4]
```diff
@@ -9,8 +9,6 @@ namespace Microsoft.PowerShell.Commands
     using System.Management.Automation.Host;
     using System.Management.Automation.Internal;
     using Microsoft.PowerShell.Commands.Internal.Format;
-
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs]: Put back this whitespace

File: src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs [hunk 5]
```diff
@@ -23,8 +21,6 @@ public class OutNullCommand : PSCmdlet
         /// </summary>
         [Parameter(ValueFromPipeline = true)]
         public PSObject InputObject { set; get; } = AutomationNull.Value;
-
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs]: Put back this whitespace

File: src/System.Management.Automation/engine/CommandProcessorBase.cs [hunk 6]
```diff
@@ -682,11 +675,9 @@ public override string ToString()
         /// This default implementation reads the next pipeline object and sets
         /// it as the CurrentPipelineObject in the InternalCommand.
         /// </summary>
-        ///
         /// <returns>
         /// True if read succeeds.
         /// </returns>
-        ///
         /// does not throw
```
Comment (SteveL-MSFT) [inline src/System.Management.Automation/engine/CommandProcessorBase.cs]: Can we move this to inside the \<summary/\>?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]
@iSazonov I appreciate your passion to clean up the code, but perhaps we should discuss whether it's worth the review effort next time before you start a. PR as this was review took significant time.

File: src/Microsoft.PowerShell.Security/security/CertificateProvider.cs (hunk 2 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.Security/security/CertificateProvider.cs]: Seems broken EOL.

Fixed.

File: src/System.Management.Automation/FormatAndOutput/common/ILineOutput.cs (hunk 3 above)
Comment (iSazonov) [reply src/System.Management.Automation/FormatAndOutput/common/ILineOutput.cs]: Fixed.

File: src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs (hunk 4 above)
Comment (iSazonov) [reply src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs]: Fixed.

File: src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs (hunk 5 above)
Comment (iSazonov) [reply src/System.Management.Automation/FormatAndOutput/out-console/OutConsole.cs]: Fixed.

File: src/System.Management.Automation/engine/CommandProcessorBase.cs (hunk 6 above)
Comment (iSazonov) [reply src/System.Management.Automation/engine/CommandProcessorBase.cs]: Fixed.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by iSazonov** [comment]
@SteveL-MSFT We already have a tracking Issue but haven't _plan_. With the plan we'd freeze current PRs and spend 1-2 weeks to fix as many style issues as possible.

**Pull request merged**